server.js

```diff
@@ -2040,6 +2040,135 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// GitHub pullrequests integration.
+camp.route(/^\/github\/issues-pr(-raw)?\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
```

Comment:
It would be great to combine all those new `camp.route` endpoints into the existing `issues` endpoint. It seems possible, and I believe it would greatly reduce the amount of code. There'd be a bit of code to extract the dashed options (`-pr`, `-raw`, …).